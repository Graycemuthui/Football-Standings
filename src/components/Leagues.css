.leagues {
  display: grid;
  grid-template-columns: repeat(2, 1fr);
  background-color: #13bdba;
}

.league {
  display: flex;
  padding: 0.2em;
  min-height: 10em;
}

.league:nth-child(4n + 2) {
  background-color: #0e19df;
}

.league:nth-child(4n + 3) {
  background-color: #0e19df;
}

.upper-league {
  display: flex;
  flex-direction: column;
  align-items: flex-end;
  justify-content: space-between;
}

.league-name {
  align-self: flex-end;
  color: rgb(0, 0, 0);
  margin: 0.5em 0;
  min-width: 4em;
  font-size: 1.1rem;
}

.image {
  max-width: 90%;
}

.league-btn {
  border: none;
  cursor: pointer;
}

h2 {
  color: #fff;
  background-color: #0e19df;
  margin: 0;
  font-size: 1.1rem;
  padding: 1rem;
}
