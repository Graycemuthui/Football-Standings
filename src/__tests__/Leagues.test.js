import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/ConfigureStore';
import Leagues from '../components/Leagues';

describe('Leagues test', () => {
  it('leagues render correctly', () => {
    const app = renderer
      .create(
        <Provider store={store}>
          <Leagues />
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
