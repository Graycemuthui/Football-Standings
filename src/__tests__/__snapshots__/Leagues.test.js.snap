// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Leagues test leagues render correctly 1`] = `
<div
  className="grid-container"
>
  <div>
    <h2
      className="heading-text"
    >
      International leagues
    </h2>
  </div>
  <div
    className="leagues"
  />
</div>
`;
